pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -33,6 +34,10 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 )
 
+// Changes outside of the kubernetes universe could cause the approval decision
```

Comment:
More like outside the state of this controller. Adding a new RBAC role binding is not exactly outside the Kubernetes universe.